modules/ROOT/pages/organization-management-in-bonita-bpm-studio.adoc

```diff
@@ -74,7 +74,7 @@ The organization that you deploy overwrites one in the Bonita Runtime.
 [WARNING]
 ====
 
-Importing an organization overwrites the existing organization in Bonita Portal, which could have an impact on deployed processes.
+Importing an organization overwrites the existing organization in Bonita Runtime, which could have an impact on deployed processes.
```

Comment:
Runtime, which could have an impact on deployed processes. => Runtime, and this could have an impact on deployed processes. Take the time to check and test those processes in order to avoid regressions.

Reply:
Done